README.md

```diff
@@ -17,10 +17,10 @@ Build scripts to help generate cross ToolChains for various platforms supported
 sudo pacman -S base-devel gperf help2man bison texinfo flex gawk
 ```
 
-## For Ubuntu users:
+## For Ubuntu/Debian users:
```

Comment:
Somehow that looks odd; I'm used to Debian/Ubuntu. :-P

Reply:
I just added it the end, but now that you mention it, it does look odd ...